registrations/notifications.py

```diff
@@ -0,0 +1,249 @@
+            ),
+            str(Event.TypeId.COURSE): _(
+                "You will be automatically transferred as a course participant if a seat becomes available."
+            ),
+            str(Event.TypeId.VOLUNTEERING): _(
+                "You will be automatically transferred as a volunteering participant if a seat becomes available."
+            ),
+        },
+        "group": {
+            "text": {
+                str(Event.TypeId.GENERAL): _(
+                    "The registration for the event <strong>%(event_name)s</strong> waiting list was successful."
+                ),
+                str(Event.TypeId.COURSE): _(
+                    "The registration for the course <strong>%(event_name)s</strong> waiting list was successful."
+                ),
+                str(Event.TypeId.VOLUNTEERING): _(
+                    "The registration for the volunteering <strong>%(event_name)s</strong> waiting list was successful."
+                ),
+            },
+            "secondary_text": {
+                str(Event.TypeId.GENERAL): _(
+                    "You will be automatically transferred from the waiting list to become a participant in the event if a place becomes available."  # noqa E501
+                ),
+                str(Event.TypeId.COURSE): _(
+                    "You will be automatically transferred from the waiting list to become a participant in the course if a place becomes available."  # noqa E501
+                ),
+                str(Event.TypeId.VOLUNTEERING): _(
+                    "You will be automatically transferred from the waiting list to become a participant in the volunteering if a place becomes available."  # noqa E501
+                ),
+            },
+        },
+    },
+    SignUpNotificationType.TRANSFERRED_AS_PARTICIPANT: {
+        "heading": _("Welcome %(username)s"),
+        "text": {
+            str(Event.TypeId.GENERAL): _(
+                "You have been moved from the waiting list of the event <strong>%(event_name)s</strong> to a participant."  # noqa E501
+            ),
+            str(Event.TypeId.COURSE): _(
+                "You have been moved from the waiting list of the course <strong>%(event_name)s</strong> to a participant."  # noqa E501
+            ),
+            str(Event.TypeId.VOLUNTEERING): _(
+                "You have been moved from the waiting list of the volunteering <strong>%(event_name)s</strong> to a participant."  # noqa E501
+            ),
+        },
+    },
+}
+
+
+def get_signup_notification_texts(signup, notification_type: SignUpNotificationType):
+    with (translation.override(signup.get_service_language_pk())):
+        confirmation_message = signup.registration.confirmation_message
+        event_name = signup.registration.event.name
+        event_type_id = signup.registration.event.type_id
+        username = signup.first_name
```

Comment:
```suggestion

    event_type_id = signup.registration.event.type_id
    username = signup.first_name
```
These don't need to be translated if I've understood everything correctly.

Reply:
True that. Moved those variables outside of the `with translation.override...`